app/models/food_item.py

```diff
@@ -0,0 +1,16 @@
+'''
+FoodItem stores all food item related data
+such as station, name, calories, categories, etc.
+'''
+class FoodItem:
```

Comment:
Do we also want to include which restaurant the food item belongs to?

Reply:
Yes, good idea -- patching in another branch